lib/hcc-config.cmake.in

```diff
@@ -22,6 +24,16 @@ set_target_properties(hsa-runtime64 PROPERTIES
   INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${HSA_HEADER}"
 )
 
+pkg_check_modules(LIBUNWIND REQUIRED libunwind)
+add_library(${LIBUNDWIND_LIBRARIES} INTERFACE IMPORTED)
```

Comment:
thanks for the explanation. would it make sense to move those targets to common ImportTargets.cmake to be used by both build and downstream?